Fix multiple re-syncing

stephanheilner opened the pull request.

nickmshelley commented:
👍  Thanks for fixing my crap! 😄

hiltonc commented on LDSAnnotationsTests/SessionNotificationsTests.swift:
```diff
@@ -30,29 +30,6 @@ class SessionNotificationsTests: XCTestCase {
         XCTAssertEqual(session.status, Session.Status.Unauthenticated)
     }
     
-    func testAuthSuccessNotifications() {
```
We dropped a test here, would it make sense to replace it, or is it no longer needed?

stephanheilner commented:
The two that I dropped we to test the status notifications when sign in fails and succeeds. Since we don't update the status for those anymore, the tests wouldn't be helpful. If we add an "authStatus" with observers, then we'll want them again.

> On Oct 8, 2016, at 9:37 AM, Hilton Campbell <notifications@github.com> wrote:
> 
> @hiltonc commented on this pull request.
> 
> In LDSAnnotationsTests/SessionNotificationsTests.swift:
> 
> > @@ -30,29 +30,6 @@ class SessionNotificationsTests: XCTestCase {
>          XCTAssertEqual(session.status, Session.Status.Unauthenticated)
>      }
>      
> -    func testAuthSuccessNotifications() {
> We dropped a test here, would it make sense to replace it, or is it no longer needed?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

hiltonc commented:
OK, cool. Thanks!
